internal/testbed/load/tests/metrics_test.go

```diff
@@ -247,6 +247,30 @@ func TestPrometheusMetric(t *testing.T) {
 			},
 			processors: defaultProcessors,
 		},
+		{
+			name:     "Prometheus Prometheus 1kDPS - 10 Prometheus Endpoints",
+			sender:   datasenders2.NewMultiHostPrometheusDataSender(testbed.DefaultHost, testutil.GetAvailablePorts(t, 10)),
+			receiver: testbed.NewOTLPDataReceiver(testutil.GetAvailablePort(t)),
+			extendedLoadOptions: ExtendedLoadOptions{
+				resourceSpec: testbed.ResourceSpec{
+					ExpectedMaxCPU: 500,
```

Comment:
these limits are set relatively high for now, as the load tests are executed on the main branch only by now. Once this PR has been merged, I will do a follow up PR to change them according to the actual resource consumption